### PR TITLE
Added the description text file.

### PR DESCRIPTION
**Pull request by AsrarulHoque**
Added the text file for adding detailed description of the project.

### CODE REVIEW

File: text.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Hey We are practicing git and github.
```
Comment (academy-yousuf) [inline text.txt]: Separate them in to different lines.

**Review by academy-yousuf — CHANGES_REQUESTED** [review]
Follow the comments and make appropriate adjustments.

File: text.txt (hunk 1 above)
Comment (AsrarulHoque) [reply text.txt]: Separated into different lines.

**Review by academy-yousuf — APPROVED** [review]
LG...

**Pull request merged**